src/stellar-wallet.tsx

```diff
@@ -13,10 +13,12 @@ export type Wallet = {
   displayName: string;
 };
 
-export type Balance = StellarSdk.Horizon.HorizonApi.BalanceLine;
+export type Balance =
+  | { trustline: false }
+  | { trustline: true; balanceLine: StellarSdk.Horizon.HorizonApi.BalanceLine };
```

Comment:
This type could be something like which might be a tad friendlier to use, as the balanceLine cannot be undefined 🤔 

```export type Balance =
  | { trustline: false, balanceLine: { balance: "0" } }
  | { trustline: true; balanceLine: StellarSdk.Horizon.HorizonApi.BalanceLine };
  ```